src/utils/wallet.js

```diff
@@ -667,7 +667,7 @@ class Wallet {
                 await this.postSignedJson('/account/validateSecurityCode', body);
             }
         } catch(e) {
-            throw new WalletError('Invalid code', 'setupRecoveryMessage.error')
+            throw new WalletError('Invalid code', 'setupRecoveryMessageNewAccount.invalidCode')
```

Comment:
@marcinbodnar let's have a check for specific error from contract helper. I believe it provides 401 code + messages on what exactly happened:
https://github.com/near/near-contract-helper/blob/f16b2f8d5d691c4174e5ee33636005f735775220/middleware/2fa.js#L243

Reply:
@vgrichina this error is for email and phone recovery code, so it's this one in CH:
https://github.com/near/near-contract-helper/blob/f16b2f8d5d691c4174e5ee33636005f735775220/app.js#L291

But it throws `recoveryMethod does not exist` which doesn't seem to be relevant. Want me to check this message string anyway?

Reply:
@marcinbodnar ah, got it

it is relatively relevant, think of it same as when validating user/password you usually just report that there is no user with such password (vs revealing that user exists)

not sure how important though, maybe we should give more granular info

but I think we need to catch this specific error if we are only expecting it

Reply:
@marcinbodnar anyway, gonna merge – let's fix in follow-up.

Reply:
@vgrichina 
> it is relatively relevant, think of it same as when validating user/password you usually just report that there is no user with such password (vs revealing that user exists)

Right, got it. I will add this check in my next PR.